packages/datetime/src/common/classes.ts

```diff
@@ -27,29 +27,17 @@ export const DATE_INPUT_TIMEZONE_SELECT = `${NS}-date-input-timezone-select`;
 
 export const DATEPICKER = `${NS}-datepicker`;
 export const DATEPICKER_CAPTION = `${DATEPICKER}-caption`;
-export const DATEPICKER_CAPTION_CARET = `${DATEPICKER_CAPTION}-caret`;
 export const DATEPICKER_CAPTION_MEASURE = `${DATEPICKER_CAPTION}-measure`;
-export const DATEPICKER_CAPTION_SELECT = `${DATEPICKER_CAPTION}-select`;
 export const DATEPICKER_CONTENT = `${DATEPICKER}-content`;
-export const DATEPICKER_DAY = "DayPicker-Day";
```

Comment:
Any classes with `"DayPicker-Day"` as the prefix can safely be eliminated. This refers to the styling methodology used by `react-day-picker` v7.